### PR TITLE
Simpler hex

### PR DESCRIPTION
**Pull request by bukzor**
I couldn't think of better function names -.-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Upstream's not accepting PRs, let's fork!
 
 Things you'll need beforehand:
-- A pokemon.com account (I'd suggest making an account separate from the one
+- A pokemon.com account (I'd suggest [making an account][3] separate from the one
   you play on since this probably violates the TOS).
+  You can create an account from your desktop at
```
Comment (asottile) [inline README.md:6]: Oops

File: scrape.py [hunk 2]
```diff
@@ -118,6 +118,7 @@ def get_api_endpoint(access_token, lat_i, lng_i):
             print('retrying_get_profile: get_profile returned no api_url, retrying')
             profile_response = None
             continue
+        print(profile_response)
```
Comment (asottile) [inline scrape.py:121]: Remove prints plz?

**Comment by asottile** [comment]
Fixnship.  Want to regen the protobuf python or is it not necessary

**Comment by bukzor** [comment]
I won't get to this soon

**Comment by asottile** [comment]
Continued in #16

**Pull request closed without merging**